org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/variables/SetValueFunction.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
```

Comment:
use SetValueFunction to allow unchecked exception InvalidTypeException, ClassNotLoadedException